docs/source/technical_tutorials/working_with_r/gedi_access_r.ipynb

```diff
@@ -0,0 +1,906 @@
+{
```

Comment:
<p>This is a very python way of doing this. I'll come back in a few min with the R way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>